### ISSUE TITLE
Position fixed leads to gaps in the comment stream

### ISSUE THREAD

**Issue by oliver-dvorski** [coralproject/talk#4371]
**Expected behavior:**
Comments should scroll normally, like when the position of the container is set to `relative`

**Actual behavior:**
Setting `position: fixed` on a parent of the comment stream, leads to glitches in the `virtuoso-item-list`. This way the user can't see all comments.

![image](https://github.com/coralproject/talk/assets/9038049/31c6ebf7-e7e8-423a-900f-5de48ee4a22a)

**Related Issues:** 
None

**Versions:**
Coral Talk 8.5.3

- NodeJS: 14 (using the provided Dockerfile)
- NPM: 8.0.0
- MongoDB: 4.2
- Redis: 3.2
- Browser: Firefox 118
- OS: Arch Linux

https://youtu.be/gU-BARBHmrk

**Comment by tessalt** [comment]
Hi there, embedding coral within a fixed position container is a bit of a special case but it can be achieved with the correct CSS. We can't debug CSS issues on your site, but you can take a look at examples such as www.theverge.com to see how this can be accomplished.

**Comment by oliver-dvorski** [comment]
Thank you for your reply!

Really, I do appreciate the time you're putting into reading these issues. I agree that a CSS issue specific to our site wouldn't be your concern and that you couldn't offer support for that. That's perfectly reasonable. However, I used a blank site in the video and screenshot provided in my initial bug report, thereby eliminating any custom CSS, other than the fixed positioning of the stream container.

Based on this, I believe that the occurrence of this error on a pristine site, solely due to placing the comment stream within a `position: fixed` container, is likely an issue on Coral's end. At the very least, it seems unpredictable from a user's perspective. If we classify this as a unique case, I kindly request your assistance in finding a workaround. Once found, we can document it so as  to benefit the entire community. I've noticed that your documentation already addresses similar limitations when it comes to [Virtuoso](https://virtuoso.dev/), so it would be great to have this issue covered as well.

Thank you for providing The verge as an example. Indeed, I've been trying to figure out how they worked around this for days. Since Coral is hosting their instance of Talk themselves, I figured you might already have the answer. So far I've been able to eliminate the following approaches:

- **[Their Coral style sheet](https://www.theverge.com/_next/static/coral.061b36318fd93ea4499d962535a92c95.css)**. When including this style sheet in my local coral instance (same version as theirs, 8.5.3), the bug is still there
- **Their page styles.** I've loaded their CSS into my otherwise blank testing site and the bug is still there
- **Their use of nesting `relative` and `fixed` elements higher up in the hierarchy of the comment stream**. I've copied their HTML from the Headless UI Portal Root to the coral container, with their styles and without them, and the bug is still there.

Again, thank you for your time and consideration on this matter. Hopefully we can find and document a workaround for this, for the rest of the community to benefit from.
